src/SimpleHypergraphs.jl

```diff
@@ -1,8 +1,13 @@
 module SimpleHypergraphs
 
+using LightGraphs
+
 export Hypergraph, getvertices, gethyperedges, hg_load, hg_save
+export add_vertex!, add_hyperedge!
+export BipartiteView, shortest_path
 
 include("types.jl")
+include("bipartite.jl")
```

Comment:
typically it is better to keep types in one file as it is easier to manage code later. but we can go the way you propose, then `types.jl` file should be renamed.